benchcab/internal.py

```diff
@@ -57,23 +57,27 @@
     CABLE_AUX_DIR / "core" / "biogeochem" / "pftlookup_csiro_v16_17tiles.csv"
 )
 
+# Fluxsite directory tree
+FLUXSITE_DIRS = {}
```

Comment:
Would a list be better here? I.e.
```python
FLUXSITE_DIRS = [
    FLUXSITE_RUN_DIR,
    FLUXSITE_LOG_DIR,
    ...
]
```
I don't mind either way, but it seems using a list would have less changes in the code base.

Reply:
I don't like having 2 places with the same constants: the constant with its name and then a list with all of them that is used only once in the code after that. Hence the idea of a dictionary so that references to these paths are always handled in the same way throughout the code.